include/openspace/properties/numericalproperty.inl

```diff
@@ -61,6 +61,7 @@ T NumericalProperty<T>::minValue() const {
 
 template <typename T>
 void NumericalProperty<T>::setMinValue(T value) {
+    Property::notifyMetaDataChangeListeners();
```

Comment:
```suggestion
    notifyMetaDataChangeListeners();
```

Reply:
I'm afraid I can't do that 
![image](https://github.com/user-attachments/assets/58479e59-4a61-400e-a55d-0301a1ff9d2a)